packages/support/src/Commands/CheckMissingTranslationsCommand.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Filament\Support\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Filesystem\Filesystem;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Collection;
+use Symfony\Component\Console\Exception\InvalidOptionException;
+use Symfony\Component\Finder\SplFileInfo;
+use function Termwind\{render};
+
+class CheckMissingTranslationsCommand extends Command
+{
+    protected $signature = 'filament:check-translations
+                            {lang*              : The lang to compare against the base translation. Accepts multiple values.}
+                            {--B|baseLang=en    : The base lang to compare with.}
+                            {--D|dir=app        : The directory containing the translations. \'app\' and \'vendor\' are accepted values.}';
+
+    protected $description = 'Scan translation files and compare each of them with the base translation. Outputs missing and deprecated translations.';
+
+    public function handle()
+    {
+        $this->scan('filament');
+        $this->scan('forms');
+        $this->scan('support');
+        $this->scan('tables');
+
```

Comment:
The notifications package is missing from this list. There's translations for notifications as well since database notifications got introduced.